sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -139,13 +139,18 @@ def _decode_error(response, error_message=None, error_type=None, **kwargs):  # p
                     additional_data[info.tag] = info.text
         else:
             if error_body:
-                for info in error_body.iter():
-                    if info.tag.lower().find("code") != -1:
-                        error_code = info.text
-                    elif info.tag.lower().find("message") != -1:
-                        error_message = info.text
-                    else:
-                        additional_data[info.tag] = info.text
+                try:
+                    # Attemp to parse from XML element
+                    for info in error_body.iter():
+                        if info.tag.lower().find("code") != -1:
+                            error_code = info.text
+                        elif info.tag.lower().find("message") != -1:
+                            error_message = info.text
+                        else:
+                            additional_data[info.tag] = info.text
+                except:
```

Comment:
Sorry my fault - we should avoid a bare except (this will probably also need a pylint disable):
```suggestion
                except Exception:
```